lib/Importer/Caffe2.cpp

```diff
@@ -270,9 +270,17 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
 
     Node *node = G_.createConcat(opName, inputs, channel);
 
+    // The semantic of the C2 concat operator is a bit different
```

Comment:
Concat in C2 has 2 modes. Do this new code path when `add_axis=1`, otherwise old implementation is correct.

Reply:
Good point I haven’t!
And I managed to screw the writing of the test.

Reply:
Done and added a test for each case.